rest/OrderService.js

```diff
@@ -56,12 +60,27 @@ class OrderService {
             throw new IllegalArgumentError(EMPTY_ARGUMENT_ERR);
         }
     }
+    
+    #checkProductIsActive(productId) {
```

Comment:
assertProductIsActive

Pretty much make it the reverse method of assertProductIsNotActive